RevenueCatUI/Templates/V2/Components/Stack/StackComponentView.swift

```diff
@@ -91,7 +91,11 @@ struct StackComponentView: View {
                       verticalAlignment: verticalAlignment.frameAlignment)
             case .zlayer(let alignment):
                 ZStack(alignment: alignment.stackAlignment) {
-                    ComponentsView(componentViewModels: self.viewModel.viewModels, onDismiss: self.onDismiss)
+                    ComponentsView(
+                        componentViewModels: self.viewModel.viewModels,
+                        ignoreSafeArea: self.viewModel.shouldApplySafeAreaInset,
```

Comment:
The parameter name `ignoreSafeArea` seems contradictory with sending `self.viewModel.shouldApplySafeAreaInset`